src/index.js

```diff
@@ -1,2 +1,7 @@
-export {default as RandomForestClassifier} from './RandomForestClassifier';
-export {default as RandomForestRegression} from './RandomForestRegression';
+import {default as RandomForestClassifier} from './RandomForestClassifier';
+import {default as RandomForestRegression} from './RandomForestRegression';
+
+export {
```

Comment:
mmmm ok, I think that I should avoid that default because that cause problems with rollup, thank you